02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -6,3 +6,61 @@ output:
     toc_float: true
 ---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to Microarray technology](#introduction-to-microarray-technology)
+  - [Microarray data **strengths**:](#microarray-data-strengths)
+  - [Microarray data **limitations**:](#microarray-data-limitations)
+- [About quantile normalization](#about-quantile-normalization)
+  - [More resources on microarray technology:](#more-resources-on-microarray-technology)
+    - [Why doesn't the gene I care about show up in this microarray dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-microarray-dataset)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to Microarray technology
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
+As with all experimental methods, microarray has strengths and limitations that you should consider in regards to your scientific questions. 
+
+### Microarray data **strengths**:  
+
+- Less expensive than RNA-seq - can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray has a faster turn-around than RNA-seq [@intro-microarray-slides].
+- It's a simpler experimental procedure than RNA-seq. 
+
+### Microarray data **limitations**:  
+
+- If a transcript doesn't have a probe designed to it, you won't get it measured; can't be used for transcript discovery [@Mantione2014]. 
+- A chip's probe designs are only as up to date as the genome annotation at the time it was designed [@Mantione2014].
+- As is true for all techniques that involve nucleotide hybiridization (RNA-seq too); microarray probes come with some biases depending on their nucleotide sequence composition. 
+
+To help address these within-chip probe biases, refine.bio uses [SCAN](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508193/pdf/nihms401888.pdf) normalization methods [@Piccolo2012]
+
+## About quantile normalization 
+
+Microarray data is generally processed in groups of chips - this can lead to [batch effects](https://en.wikipedia.org/wiki/Batch_effect#:~:text=In%20molecular%20biology%2C%20a%20batch,of%20interest%20in%20an%20experiment). 
+To minimize this, all refine.bio microarray data downloads come [quantile-normalized](https://en.wikipedia.org/wiki/Quantile_normalization). 
+See the refine.bio docs for more about the microarray process steps including the [quantile normalization](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization).
+
```

Comment:
I've debated about adding a `limma` package section here but we don't use it the same way we use `DESeq2` in RNA-seq. And I'm not really sure what to say about limma. It's only used in the two differential expression modules and none of the rest.

Reply:
That is fine with me. I think that makes sense.